app/src/test/java/org/dhis2/usescases/enrollment/EnrollmentPresenterImplTest.kt

```diff
@@ -167,45 +181,13 @@ class EnrollmentPresenterImplTest {
         verify(valueStore, times(1)).save("uid", "fileValue")
     }
 
-    @Test
-    fun `Check data integrity when mandatory is false and has error is false`() {
-        val emptyFields: List<String> = listOf("field")
-        val errorFields: List<String> = listOf("field")
-        val result = presenter.dataIntegrityCheck(emptyFields, errorFields)
-        verify(enrollmentView, times(1)).showMissingMandatoryFieldsMessage(emptyFields)
-        verify(enrollmentView, times(0)).showErrorFieldsMessage(errorFields)
-        Assert.assertFalse(result)
-    }
-
     @Test
     fun `Check data integrity when mandatory is true and has error is false`() {
-        val emptyFields: List<String> = listOf()
-        val errorFields: List<String> = listOf()
-        val result = presenter.dataIntegrityCheck(emptyFields, errorFields)
+        val result = presenter.dataIntegrityCheck()
         verifyZeroInteractions(enrollmentView)
         Assert.assertTrue(result)
     }
 
-    @Test
-    fun `Check data integrity when mandatory is false and has error is true`() {
-        val emptyFields: List<String> = listOf("field")
-        val errorFields: List<String> = listOf("field")
-        val result = presenter.dataIntegrityCheck(emptyFields, errorFields)
-        verify(enrollmentView, times(1)).showMissingMandatoryFieldsMessage(emptyFields)
-        verify(enrollmentView, times(0)).showErrorFieldsMessage(errorFields)
-        Assert.assertFalse(result)
-    }
-
-    @Test
-    fun `Check data integrity when mandatory is true and has error is true`() {
-        val emptyFields: List<String> = listOf()
-        val errorFields: List<String> = listOf("field")
-        val result = presenter.dataIntegrityCheck(emptyFields, errorFields)
-        verify(enrollmentView, times(0)).showMissingMandatoryFieldsMessage(emptyFields)
-        verify(enrollmentView, times(1)).showErrorFieldsMessage(errorFields)
-        Assert.assertFalse(result)
-    }
```

Comment:
this is not being test now? :(

Reply:
They are duplicated tests. The previous tests already did that.